src/elasticsearch/elasticsearchutils.py

```diff
@@ -43,22 +47,28 @@
   }
  },
  "script" : {
-  "inline" : "ctx._source.status='<STATUS>' ;ctx._source.info='<INFO>'; ctx._source.reason='<REASON>'; ctx._source.modified_at='<MODIFIED_AT>'"
+  "source" : "ctx._source.status=params.status ;ctx._source.info=params.info; ctx._source.reason=params.reason; ctx._source.modified_at=params.modified",
+  "params" : {
+    "status": '<STATUS>',
+    "modified": '<MODIFIED_AT>',
+    "info": '<INFO>',
+    "reason": '<REASON>'
+  }
  }
 }
- 
+
```

Comment:
why this line here?

Reply:
I think this is a good change to remove unnecessary whitespace from the current code.